src/main/js/main_html5.js

```diff
@@ -936,13 +938,46 @@ require("../../../html5-common/js/utils/environment.js");
           }
         }
       }
-      
+
       // audioTracks right now is Array-like, not actually an array
       // so we need to make it so
       var newTracks = this.getAvailableAudio();
       raiseAudioChange(newTracks);
     };
 
+    /**
+     * Set the playback speed of the video element
+     * @public
+     * @method OoyalaVideoWrapper#setPlaybackSpeed
+     * @param  {number} speed The desired speed multiplier
+     */
+    this.setPlaybackSpeed = function(speed) {
```

Comment:
Can you update the video plugin template with these new public functions? The template acts as part of our documentation for video plugins

Reply:
Will do.

Reply:
Added